stanford/roles/cluster/defaults/main.yml

```diff
@@ -249,33 +249,29 @@ CLUSTER_ELBS:
             cidr_ip: '0.0.0.0/0'
       subnets:
         - "{{ COMMON_DEPLOYMENT }}-subnet-edxapp-{{ CLUSTER_NUMBER }}"
-  elasticsearch_forum:
-    - name: "{{ COMMON_DEPLOYMENT }}-elb-elasticsearch_forum-private"
+  esforum:
```

Comment:
Unless we shortened this,
it made the ELB names too long.
Too long, as in, AWS won't create it, not just that it's verbose.